### ISSUE TITLE
Create pagination component

### ISSUE THREAD

**Issue by jossmac** [elementalui/elemental#9]

**Comment by grabbou** [comment]
I would just go with a `ButtonGroup` or `Toolbar` component (similar to Bootstrap one) so it can actually be used elsewhere pretty easily.

**Comment by jossmac** [comment]
Hi @grabbou,

We have a ButtonGroup, but it's more for grouping similar tools. This component will provide support for pagination and will very much be modelled on the bootstrap pagination http://getbootstrap.com/components/#pagination.

It will accept the object that express pagination returns:
```
pagination: React.PropTypes.shape({
  currentPage: React.PropTypes.number,
  first:       React.PropTypes.number,
  last:        React.PropTypes.number,
  next:        React.PropTypes.bool,
  pages:       React.PropTypes.array,
  previous:    React.PropTypes.bool,
  total:       React.PropTypes.number,
  totalPages:  React.PropTypes.number
})
```

**Comment by jossmac** [comment]
Implemented by 2aed2fa
